driver/makedepend.ml

```diff
@@ -17,6 +17,47 @@ open Compenv
 open Parsetree
 module String = Misc.Stdlib.String
 
+module Json = struct
```

Comment:
I would move this module to `utils/misc.ml`, because the logic is not generic to makedepend and could be useful in other places eventually.

Reply:
@gasche  could you please help me with this,
I am always getting the warnings like
`Error (warning 60): unused module Json.`
`Error (warning 32): unused value print.` ...

Reply:
@muskangarg21 you probably need to export the interface of the Json module in `misc.mli`. If you don't do this, then the module is not accessible from the outside and the code cannot be used, which would explain the warnings you see.